addons/packages/velero/1.7.1/package.yaml

```diff
@@ -0,0 +1,415 @@
+                  default: "default"
+            configAzure:
+              type: object
+              description: >
+                Configuration specific to the Azure plugin
+              properties:
+                apiTimeout:
+                  type: string
+                  description: >
+                    Indicates how long to wait for an Azure API request to complete before timeout. Defaults to 2m0s.
+                  default: "2m0s"
+                resourceGroup:
+                  type: string
+                  description: >
+                    The name of the resource group where volume snapshots should be stored, if different from the
+                    cluster's resource group. Optional.
+                  default: ""
+                subscriptionId:
+                  type: string
+                  description: >
+                    The ID of the subscription where volume snapshots should be stored, if different from the cluster's
+                    subscription. Requires "resourceGroup" to also be set. Optional.
+                  default: ""
+                incremental:
+                  type: boolean
+                  description: |
+                    Azure offers the option to take full or incremental snapshots of managed disks.
+                    - Set this parameter to true, to take incremental snapshots.
+                    - If the parameter is omitted or set to false, full snapshots are taken (default).
+                    Optional.
+                  default: false
+        rbac:
+          type: object
+          description: Role based access controls for Velero.
+          properties:
+            create:
+              type: boolean
+              description:
+              default: false
+            clusterAdministrator:
+              type: boolean
+              description:
+              default: false
+            name:
+              type: string
+              description: The name of the cluster role binding.
+              default: velero
+            roleRefName:
+              type: string
+              description: The name of the role.
+              default: velero
+            clusterRoleAPIGroups:
+              type: array
+              description: Array of the cluster role API groups.
+              items:
+                type: string
+            clusterRoleVerbs:
+              type: array
+              description: Array of the cluster role verbs.
+              default: ["get", "watch", "list"]
+              items:
+                type: string
+        serviceAccount:
+          type: object
+          description: Information about the Kubernetes service account Velero uses.
+          properties:
+            name:
+              type: string
+              description: The name for the service account
+              default: velero
+            annotations:
+              type: object
+              additionalProperties: string
```

Comment:
```suggestion
              additionalProperties:
                type: string
```

Reply:
Thanks. It's reasonable to use string map to represent the annotations.